src/pluto/pluto.py

```diff
@@ -509,6 +509,57 @@ def create_expected(df, v1, v2):
                     st.markdown(f"* in {v2} but not in {v1}:")
                     st.write(in2not1)
 
+
+    def create_outlier(df, v1, v2, condo, mapped):
+        outlier = df.loc[
+            (df.condo == condo)
+            & (df.mapped == mapped)
+            & (df.v == v1),
+            :,
+        ]
+
+        outlier_records = outlier.to_dict("records")
+        v1_outlier = [i["outlier"] for i in outlier_records if i["v"] == v1][0]
+
+        def display_dataframe(v1_outlier, field):
```

Comment:
I think as constructed this function is probably more aptly named 
``filter_records_by_field`` or something like that.